compiler/rustc_error_messages/locales/en-US/interface.ftl

```diff
@@ -41,3 +41,6 @@ interface_rustc_error_unexpected_annotation =
 
 interface_failed_writing_file =
     failed to write file {$path}: {$error}"
+
+interface_proc_macro_crate_panic_abort =
+    building proc macro crate with `panic=abort` may crash the compiler should the proc-macro panic
```

Comment:
```suggestion
    building a proc macro crate with `panic=abort` may crash the compiler should the proc-macro panic
```

Reply:
I used that wording due to the same being used for the following error/warning

> interface_proc_macro_doc_without_arg =
>    Trying to document proc macro crate without passing '--crate-type proc-macro to rustdoc
> .warn = The generated documentation may be incorrect

I am fine with changing this, but should that message there also be changed for consistency?

Reply:
I was kind of on the fence with this suggestion anyway, and I think consistency with the other messages is a good reason to stick with your original wording. Let's leave it as is.